src/file/file.repository.ts

```diff
@@ -1,7 +1,10 @@
 import { ObjectID } from 'mongodb';
-import { IFile } from './file.interface';
-import { fileModel } from './file.model';
+import { IFile, IPopulatedShortcut, IShortcut, PrimitiveFile } from './file.interface';
+import { baseFileModel, fileModel, shortcutModel } from './model';
+import { getCurrTraceId, log, Severity } from '../utils/logger';
 import { FileNotFoundError } from '../utils/errors/client.error';
+import { getFailedMessage } from './model/config';
+import { FileService } from "./file.service";
```

Comment:
The purpose of moving the functionality up to the service is not to call it from the repository. The repository should not know about the above layers. If you decided to move the functionality to the service, move all the usage of it up too. You can't do half here and half there.

Reply:
done, i fixed the whole functionality and moved what ever need (in repo file there's no use of FileService) and i moved the fileFactory downwards to the repo so it'll be used downwards and not upwards for the functionality stuff.

Reply:
about the getFailedMessage, some of the functions that use the logger in th repo are being used in different files like: upload.repository and not in the file.service so idk if you want me to move to the service or do anything with it.